Cargo.toml

```diff
@@ -22,7 +22,7 @@ members = [
     "examples/supergraph-sdl/rust",
     "examples/jwt-auth/rust",
     "fuzz",
-    "xtask",
+    # Note that xtask is not in the workspace member because it relies on dependencies that are incompatible with the rotuer. Notibly hyperx but there are others.
```

Comment:
```suggestion
    # Note that xtask is not in the workspace member because it relies on dependencies that are incompatible with the router. Notably hyperx but there are others.
```